config/crd/kustomization.yaml

```diff
@@ -2,34 +2,36 @@
 # since it depends on service name and namespace that are out of this kustomize package.
 # It should be run by config/default
 resources:
-- opensearch/opensearch.opster.io_opensearchclusters.yaml
-- grafana/grafana.opni.io_grafanadashboards.yaml
-- grafana/grafana.opni.io_grafanadatasources.yaml
-- grafana/grafana.opni.io_grafanas.yaml
-- bases/ai.opni.io_opniclusters.yaml
-- bases/ai.opni.io_pretrainedmodels.yaml
-- bases/core.opni.io_bootstraptokens.yaml
-- bases/core.opni.io_keyrings.yaml
-- bases/core.opni.io_gateways.yaml
-- bases/core.opni.io_loggingclusters.yaml
-- bases/core.opni.io_monitoringclusters.yaml
-- bases/core.opni.io_alertingclusters.yaml
-- bases/core.opni.io_natsclusters.yaml
-#- bases/core.opni.io_collectors.yaml
-- bases/logging.opni.io_datapreppers.yaml
-- bases/logging.opni.io_logadapters.yaml
-- bases/logging.opni.io_loggingclusterbindings.yaml
-- bases/logging.opni.io_multiclusterrolebindings.yaml
-- bases/logging.opni.io_multiclusterusers.yaml
-- bases/logging.opni.io_opniopensearches.yaml
-- bases/logging.opni.io_opensearchrepositories.yaml
-- bases/logging.opni.io_snapshots.yaml
-- bases/logging.opni.io_recurringsnapshots.yaml
-#- bases/logging.opni.io_collectorconfigs.yaml
-- bases/monitoring.opni.io_rolebindings.yaml
-- bases/monitoring.opni.io_roles.yaml
-#- bases/monitoring.opni.io_collectorconfigs.yaml
-- bases/logging.opni.io_preprocessors.yaml
+  - opensearch/opensearch.opster.io_opensearchclusters.yaml
```

Comment:
should probably keep the indentation here for history purposes

Reply:
The indentation is auto done by the editor.